include/webserv.hpp

```diff
@@ -2,12 +2,20 @@
 #define WEBSERV_H
 
 #include <string>
-#include "Environment.hpp"
 #include <sstream>
 #include <cstdlib>
 #include <string.h>
+#include "MetaVariables.hpp"
 #include "HTTPRequest.hpp"
 
+// #include "MetaVariables.hpp"
```

Comment:
You wanted to remove all the commented out stuff right?

Reply:
Must have got mixed up with the two branches - gone